Added RRDGLCD support for the new Watch Screen.

garikdan opened the pull request:
- Added graphics functions.
pixel
drawLine
drawHLine
drawVLine
drawBox
- Added function to set the cursor position.
setCursorPX and setCursor.
- Improved functions.
displayChar
displayString
renderChar
- Changed deg symbol.

garikdan committed:
Added new Watch Screen for RRDGLCD.

Added drawing functions for RRDGLCD.
Created a new Watch screen. "Fan Speed" is added to the "Prepare" menu to adjust the speed of the cooler. On the Watch screen displays the speed of the cooler in percent. In bitmaps.h added icons for the new Watch screen. Tested on Full Graphic smart controller 12864 (ST7920).

garikdan committed:
Revert "Added new Watch Screen for RRDGLCD."

This reverts commit fbf106ccf63c4df0e80cb96e27088646b37bb345.

garikdan committed:
Added RRDGLCD support for the new Watch Screen

Added graphics functions.
Added function to set the cursor position. Improved functions displayChar(), displayString(), renderChar().

wolfmanjm commented on src/modules/utils/panel/panels/rrdglcd/RrdGlcd.cpp:
```diff
@@ -297,6 +316,68 @@ void RrdGlcd::renderGlyph(int xp, int yp, const uint8_t *g, int pixelWidth, int
+    else
+        *byte |= mask; // set pixel
+	    // Change the dirty rectangle to account for a pixel being dirty (we assume it was changed)
+    if (startRow > y) { startRow = y; }
```
These next 4 variables appear to never be used for anything, so why waste RAM by defining them?

garikdan replied:
Removed unused variables.

wolfmanjm commented:
Thank you for doing this.
It would be nice if you could fix the indentation to comply with the smoothie standard though. There are some areas where indentation is totally incorrect.
- Thank you

garikdan committed:
Indent correction, code optimization.

Correction of indents, deletion of unused variables, deletion of an unnecessary drawLine () function.

wolfmanjm commented:
excellent thank you. i'll test this today.

garikdan committed:
Fixed of the display of characters in the progress bar.

Fixed character output, there was a problem with the background. Corrected pixel output.

garikdan commented:
**Now the conclusion as it should be:**

![img_20190228_205937](https://user-images.githubusercontent.com/9148528/53590508-f7399e80-3ba2-11e9-89fb-a1f45640effe.jpg)
![img_20190228_205853](https://user-images.githubusercontent.com/9148528/53590531-015b9d00-3ba3-11e9-8fb2-d04f4df007be.jpg)
![img_20190228_211029](https://user-images.githubusercontent.com/9148528/53590538-06b8e780-3ba3-11e9-8076-cf7e7706635b.jpg)

arthurwolf commented:
Hey there, why was this deleted ? I think it's worth keeping open for people to test ( and thanks for the code btw ).
